Fixes

drazik opened the pull request:
The initial implementation lacked some little things:

* Langs in manifest
* Hint required
* Pass settings url correctly to AppLinker
* Favicons
* Translated app name
* Correct SVG illustrations alignment
* Add smartphone platform (available soon) on last step

drazik committed:
chore: Add langs to manifest

drazik committed:
feat(HintPage): Make hint required

drazik committed:
fix(SecurityPage): Pass the settings url as string to AppLinker

Passing the URL object was working, because its toString method returns
the URL correctly. But it triggered a prop types warning in the console

drazik committed:
feat: Add favicons

drazik committed:
feat: Translate app name

drazik committed:
fix: Better svg illustrations

drazik committed:
feat(InstalledPage): Add smartphone platform

drazik committed:
feat: Add hability to customize browser names

Previously the browser names were capitalized. But sometimes we want
more control. For example « ios » can't just be capitalized, we want to
show « iOS »

ptbrowne commented on src/currentBrowser.js:
```diff
@@ -0,0 +1,15 @@
+import { detect as detectBrowser } from 'detect-browser'
+import capitalize from 'lodash/capitalize'
+import platforms from 'platforms'
```
Nit: The platforms file is not advertised as a "supported platforms" file. It is a bit dangerous since someone could add a platform there and it would automatically be understood a "supported" in the currentBrowser.js context. Ideally I think a `supported: true` attribute on the platforms side could be used. Incidentally, do you think the content of this file could be merged in platforms.js  with a name like `crossbrowser.js` ?

drazik replied:
In my mind `currentBrowser`'s role is to give us informations about the current browser (do we have an extension for it? what's its name?); and `platforms` lists supported platforms with links to the according stores (currently we have firefox and chrome, but we may have android and ios in the future). So I don't think these files should be merged, as it doesn't have the same goal. But I think renaming `platforms` to `supportedPlatforms` would be a good thing.

ptbrowne replied:
We have an extension with helpers for the platform here but I am not sure it fits the bill here: https://github.com/cozy/cozy-libs/tree/master/packages/cozy-device-helper

drazik replied:
I considered it, but here we want to have the browser's name (firefox, chrome, safari...). Maybe we can think about adding this information to what cozy-device-helper returns though.

drazik replied:
See https://github.com/cozy/cozy-libs/issues/901

ptbrowne commented on src/components/InstalledPage.jsx:
```diff
@@ -14,6 +14,19 @@ import snarkdown from 'snarkdown'
 
+const PlatformButton = props => {
+  const { icon, ...rest } = props
+  const Component = rest.href ? ButtonLink : Button
```
This feels a bit weird ? Why not using always a ButtonLink here ?

drazik replied:
Because that's a mistake, I should always use ButtonLink :) thanks

drazik committed:
refactor: Rename platforms file to supportedPlatforms

This name better reflects that this file contains informations about
supported platforms only

drazik committed:
refactor: Extract isSupportedPlatform helper
